cart/player.go

```diff
@@ -158,6 +169,21 @@ func (player *Player) Update(t int32, world *World, game *Game) {
 ///////////////////////////////////////////////////////////////////////////////
 // Utils
 
+func (player *Player) startEating(t int32) {
+	player.Eat_fx.Play()
+	player.Eating = true
+	player.EatStartTime = t
+}
+
+func (player *Player) isEating(t int32) bool {
```

Comment:
Maybe clearer to call that `IsStillEating`?

Reply:
Updated in 030ec4b to call it `updateEatingState` and not have it return a value, instead checking `player.Eating` straight after the call.